randovania/games/dread/json_data/Artaria.txt

```diff
@@ -1288,10 +1295,9 @@ Extra - asset_id: collision_camera_011
               All of the following:
                   Gravity Suit
                   Simple IBJ or Use Spin Boost
+              Movement (Intermediate) and Lay Cross Comb
```

Comment:
Could add PB + Speed Booster

Reply:
Not PBs, but you can do speedbooster yeah.